v2/car.go

```diff
@@ -44,6 +44,13 @@ type (
 	}
 )
 
+// HasIndex indicates whether the index is present.
+func (c Characteristics) HasIndex() bool {
+	// Always true because in current implementation we always write the index.
```

Comment:
I think we don't need to have this be a bit set in characteristics, and can instead use [`IndexOffset`](https://github.com/ipld/go-car/blob/cdf53fef4046264042b2fbb089de1ef1ae9e15c1/v2/car.go#L38) being non-zero and valid as an indicator of an index being present.

Reply:
sounds reasonable, I'll try and remember to note this in a spec doc